### ISSUE TITLE
Non-boost ASIO

### ISSUE THREAD

**Issue by heri16** [saleyn/eixx#46]
Now that we have #45 ...

Is it viable to remove Boost as a dependency and just use the Non-boost version of ASIO?

https://think-async.com/Asio/AsioStandalone.html

**Comment by saleyn** [comment]
I would much rather get rid of the dependency on ASIO entirely in favor of Networking TS or networking coroutines.

**Comment by heri16** [comment]
![003CA484-8A89-4990-A5D6-A95DDA4B65EF](https://user-images.githubusercontent.com/527101/130010821-8b9f72b9-bab8-47dc-8c92-31d62e40f7e5.jpeg)

Sounds good. Not familiar with this experimental API. Do you mean networking-ts-impl since some say the gcc/clang implementation is incomplete?

**Comment by saleyn** [comment]
Yes, possibly either that or implementation based on C++20 coroutines.
